library/src/CMakeLists.txt

```diff
@@ -66,6 +66,12 @@ add_library( roc::hipblas ALIAS hipblas )
 
 set(static_depends)
 
+if( WIN32 )
+  find_package( hipblas-common REQUIRED CONFIG PATHS "C:/hipSDK")
+else()
+  find_package( hipblas-common REQUIRED CONFIG PATHS /opt/rocm)
+endif()
```

Comment:
should remove hard coded C:/hipSDK but better to change both as per my earlier comment.